src/js/components/FilterByTaskState.js

```diff
@@ -58,11 +58,8 @@ class FilterByTaskState extends React.Component {
   render() {
     return (
       <Dropdown
-        buttonClassName="button dropdown-toggle text-align-left"
-        dropdownMenuClassName="
-          dropdown-menu
-          dropdown-menu-space-bottom
-          dropdown-menu-right-align"
+        buttonClassName={this.props.className}
+        dropdownMenuClassName={this.props.dropdownMenuClassName}
```

Comment:
`this.props` is usef the second time here, so you could cache it beforehands with `let {props} = this`;